docs/_includes/default.html

```diff
@@ -34,7 +34,22 @@ <h1>
       <p id="tag"><em>simple</em>, <em>flexible</em>, <em>fun</em></p>
     </header>
 
-    <main id="content">{{ content }}</main>
+    <main id="content">
+      <section>{{ content }}</section>
+
+      <section id="more-sponsors">
```

Comment:
semantically, shouldn't this be using [`<aside>`](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines#Sections_outside_the_outline)?

Reply:
I have no idea.

Reply:
rhetorical question